configure

```diff
@@ -27,7 +27,7 @@ done
 
 ## We check that $cmd is ok before the real exec $cmd
 
-`$cmd -version > /dev/null 2>&1` && exec $cmd $script "$@"
+`$cmd -version > /dev/null 2>&1` && exec $cmd -w "-3" $script "$@"
```

Comment:
It is not in general, it is just for executing `configure.ml`, and the patch is not from me but from @letouzey.